### DISCUSSION TITLE
パッケージ化に失敗する

### DISCUSSION THREAD

**Discussion by mochimochinano** [pixiv/vroid-sdk-developers#129] [Q&A]
UE5.3.2でエラーが出てパッケージ化できずに困っています。

サードパーソンテンプレートにVRM4UとVroidSDK
を入れただけの初期状態でもパッケージ化できない状態です(認証レベルと初期値のサードパーソンマップの2レベル)

・UE5.3.2
・サードパーソンテンプレート・プループリントモード/C++モード共にパッケージ化不可
・Visualstudio2022
・VroidSDK0.0.2
・VRM4U_5_3_20231126
[errorlog.txt](https://github.com/pixiv/vroid-sdk-developers/files/13575692/errorlog.txt)

エラーのログを添付します。

**Comment by ytomi-pixiv** [comment]
こんにちは、お返事が遅くなりました。

添付いただいたエラーログを拝見しました。恐らく下記リンク先の案内で解決するかと思います。
また不明点等ありましたらお知らせください。
引き続きよろしくお願いいたします。

https://ruyo.github.io/VRM4U/03_exe/#vrm4uutilactorlatest-%E3%81%AE%E3%82%A2%E3%82%BB%E3%83%83%E3%83%88%E3%81%A7%E3%82%A8%E3%83%A9%E3%83%BC%E3%81%8C%E5%87%BA%E3%82%8B

**Reply by mochimochinano** [reply]
ご教授頂きありがとうございました。
無事パッケージ化はできたのですが、完成したexeファイルを起動しようとしても全く反応せず、起動できませんでした。
セキュリティ対策ソフトを完全終了してみたのですが変わらずでした。
どういった点が問題になっていると考えられそうでしょうか。

**Comment by snghrsw** [comment]
> 無事パッケージ化はできたのですが、完成したexeファイルを起動しようとしても全く反応せず、起動できませんでした。

同バージョンにて同様の症状が発生しました。
なお、ほぼ同じ条件で VRM4Uライブラリのみを用いた場合は問題なくパッケージ化、動作確認できました。
https://github.com/ruyo/VRM4U/issues/344#issuecomment-1927253754

**Reply by snghrsw** [reply]
（追記）
- シッピングには成功するものの、.exeの実行時にウィンドウが立ち上がらないという症状が発生。
- その後、`Web Browserプラグイン` をOFFにした状態で再ビルドしたところ、シッピング・実行に成功しました。

**Comment by ytomi-pixiv** [comment]
ご報告ありがとうございます。
根本的な解決に向けて対応中ではあるものの、先ほど[0.3.1](https://github.com/pixiv/vroid-sdk-developers/discussions/138)を公開、導入マニュアルに[「配布用pkg(exe)を作成するには」](https://vroid.notion.site/VRoid-SDK-for-Unreal-Engine-21321b5e31b247e99600f270b3414d4e#ebb208652fc946cdbc9d86bcfa354ddf:~:text=%E3%81%A3%E3%81%A6%E3%81%8F%E3%81%A0%E3%81%95%E3%81%84%E3%80%82-,%E9%85%8D%E5%B8%83%E7%94%A8pkg(exe)%E3%82%92%E4%BD%9C%E6%88%90%E3%81%99%E3%82%8B%E3%81%AB%E3%81%AF,-VRM4U%E5%81%B4%E3%81%A7)を追加しました。ご確認いただけますと幸いです。
